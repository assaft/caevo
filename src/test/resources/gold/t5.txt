John left 2 days before the attack.