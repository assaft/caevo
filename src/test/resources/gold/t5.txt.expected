ei1,left
ei2,attack
t1,P2D
l1,ei1,t1,BEFORE
l2,ei2,t1,AFTER
l3,ei1,ei2,BEFORE